### PR TITLE
Autocomplete clarification + removed $

### PR DESCRIPTION
**Pull request by luckylittle**
#### What's this PR do?

* Adds clarification for SHELL autocomplete in `/content/en/docs/reference/kubectl/cheatsheet.md`

* Removes `$` in front of the commands (**BUG: [markdownlint] MD014/commands-show-output: Dollar signs used before commands without showing output**)

* Changes Markdown code reference header from `` ```console `` to `` ```bash ``

#### Where should the reviewer start?

* `cheatsheet.md`

#### How should this be manually tested?

* `echo "source <(kubectl completion bash)" >> ~/.bashrc` in BASH

* `echo "if [ $commands[kubectl] ]; then source <(kubectl completion zsh); fi" >> ~/.zshrc` in ZSH

#### Risk involved?

No

#### Screenshots (if appropriate)

N/A

### CODE REVIEW

**Comment by mistyhacks** [comment]
Deploy preview for *kubernetes-io-master-staging* ready!

Built with commit c4e68bb83d6b18ca05bcdae6331f75e00323231d

https://deploy-preview-9455--kubernetes-io-master-staging.netlify.com

**Comment by mistyhacks** [comment]
/hold

Please apply the concept template to this topic, while you are already working on it. If you notice how it is now, there is a double bullet point in the TOC: https://deploy-preview-9455--kubernetes-io-master-staging.netlify.com/docs/reference/kubectl/cheatsheet/

To find another topic with the concept template supplied, grep for `content_template: templates/concept`. You'll need to add the frontmatter and surround different parts of the file with capture tags for the overview, body, and whatsnext (if applicable).

**Comment by luckylittle** [comment]
@mistyhacks please see my commit 15b9f6a based on `/content/en/docs/concepts/example-concept-template.md`

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -5,15 +5,36 @@ reviewers:
 - erictune
 - krousey
 - clove
+content_template: templates/concept
+toc_hide: false
```
Comment (mistyhacks) [inline content/en/docs/reference/kubectl/cheatsheet.md]: nit: I think this is the default so you can remove it, but it probably won't hurt.

**Comment by mistyhacks** [comment]
/approve
/lgtm

If you want to address the nit, go for it. Otherwise you can just cancel the hold and this will go in. Thanks!

File: content/en/docs/reference/kubectl/cheatsheet.md (hunk 1 above)
Comment (luckylittle) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Yes, i just tested it and it is default. I am removing it.

**Comment by mistyhacks** [comment]
/lgtm

Thanks again!

**Pull request merged**